src/themes/OLH/templates/elements/journal/issue_list_by_decade.html

```diff
@@ -0,0 +1,24 @@
+<div class="large-12 columns">
+    <p>
+        {% blocktrans %}
+            Issues are grouped by decade. Select a decade to view the issues
+            published in during that decade.
+        {% endblocktrans %}
+    </p>
+    <ul class="inline-list no-margin-left">
+        {% for decade, issues in issues_by_decade.items %}
+            <li class="decade_list_item">
+                <a href="#decade-{{ decade }}">{{ decade }}s</a>
+            </li>
+        {% endfor %}
+    </ul>
```

Comment:
Wrapping this in a `nav` element would help screen users. Same comment for clean and material.